lib/ui/blocs/navigator/navigator_bloc.dart

```diff
@@ -144,16 +147,21 @@ class NavigatorBloc extends Bloc<NavigatorEvent, void> {
     }));
   }
 
-  void _handleNavigateToShowMnemonic() async {
+  void _handleNavigateToShowMnemonicAuth(
+      NavigateToShowMnemonicAuth event) async {
+    dynamic pushMethod = _navigatorKey.currentState.push;
+
     final method = await _getAuthenticationMethodUseCase.get();
+
     if (method is BiometricAuthentication) {
-      await _navigatorKey.currentState.push(MaterialPageRoute(
+      await pushMethod(MaterialPageRoute(
```

Comment:
As above, I think it's better to switch back this to as it was previously, to make the code easier to read

Reply:
same as above. Forgot to clean up